### PR TITLE
Take a crack at updates for v1beta1 API shape.

### PR DESCRIPTION
**Pull request by mattmoor**
Preparation for a slew of things I hope to land in 0.6

### CODE REVIEW

**Comment by mattmoor** [comment]
What version of things do the e2e tests install?  @chaodaiG @samodell

**Comment by chaodaiG** [comment]
@mattmoor , they are installing latest, and it might be missing this:
https://github.com/knative/docs/pull/1182

**Comment by chaodaiG** [comment]
@mattmoor , the installation is from here:
https://github.com/knative/test-infra/blob/master/scripts/library.sh#L322

Does your change require building serving from source?

**Comment by mattmoor** [comment]
Sweet, nightly is fine.  I just needed to know when to rerun things.  I have about 10k LoC to land before this will pass.

**Comment by chaodaiG** [comment]
lol, good to know

File: docs/serving/samples/traffic-splitting/README.md [hunk 1]
```diff
@@ -94,71 +102,47 @@ kubectl get ksvc stock-service-example --output yaml
...
    When the Service conditions report it is ready again, you can access the new
    Revision using the same method as found in the
-   [previous sample](../rest-api-go/README.md#access-the-service) by prefixing
-   the Service hostname with `latest.`.
+   [previous sample](../rest-api-go/README.md#access-the-service) using
+   the Service hostname found above.
 
 ```shell
-curl --header "Host:latest.${SERVICE_HOSTNAME}" http://${INGRESS_IP}
+curl --header "Host: ${LATEST HOSTNAME}" http://${INGRESS_IP}
```
Comment (dgerd) [inline docs/serving/samples/traffic-splitting/README.md]: This is not a real bash variable and is not defined above.

Comment (mattmoor) [reply docs/serving/samples/traffic-splitting/README.md]: I added a line that extracts the hostname.

**Comment by mattmoor** [comment]
rebased.

**Comment by matzew** [comment]
@mattmoor shoiuld this be `v1beta1` ? 
https://github.com/knative/docs/blob/master/docs/eventing/samples/cronjob-source/service.yaml#L3

**Pull request merged**